components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml

```diff
@@ -126,32 +136,17 @@
       pedigreeTrigger.observe("click", function(event) {
         event.stop();
         document.observe("xwiki:document:saved", function () {
-          window.self.location = pedigreeTrigger.href;
```

Comment:
cause it works :) I am actually wondering if there is any better way to redirect to pedigree editor. @allasm ?